tests/validators/test_decimal.py

```diff
@@ -171,24 +171,44 @@ def test_decimal_kwargs(py_and_json: PyAndJson, kwargs: dict[str, Any], input_va
 @pytest.mark.parametrize(
     'multiple_of,input_value,error',
     [
-        (0.5, 0.5, None),
-        (0.5, 1, None),
+        # Test cases for multiples of 0.5
+        *[(0.5, round(i * 0.5, 1), None) for i in range(-4, 5)],
+        (0.5, 0.49, Err('Input should be a multiple of 0.5')),
         (0.5, 0.6, Err('Input should be a multiple of 0.5')),
-        (0.5, 0.51, Err('Input should be a multiple of 0.5')),
+        (0.5, -0.75, Err('Input should be a multiple of 0.5')),
         (0.5, 0.501, Err('Input should be a multiple of 0.5')),
         (0.5, 1_000_000.5, None),
         (0.5, 1_000_000.49, Err('Input should be a multiple of 0.5')),
+        (0.5, int(5e10), None),
+        # Test cases for multiples of 0.1
+        *[(0.1, round(i * 0.1, 1), None) for i in range(-10, 11)],
         (0.1, 0, None),
-        (0.1, 0.0, None),
-        (0.1, 0.2, None),
-        (0.1, 0.3, None),
-        (0.1, 0.4, None),
-        (0.1, 0.5, None),
```

Comment:
Same question, why remove these?

Reply:
This is the same answer as https://github.com/pydantic/pydantic-core/pull/1373/files#r1687238847